README.md

```diff
@@ -1,48 +1,48 @@
 JVerein ist eine Open-Source-Vereinsverwaltung mit einer Anbindung an die ebenfalls unter Open-Source-Lizenz stehende Homebankingsoftware Hibiscus.
-Die Implementierung erfolgt mit Java. Der Ablauf auf vielen Plattformen ist damit gew�hrleistet. Als GUI-Framework kommt Jameica zum Einsatz. 
+Die Implementierung erfolgt mit Java. Der Ablauf auf vielen Plattformen ist damit gewährleistet. Als GUI-Framework kommt Jameica zum Einsatz.
 
 Mitmachen?
 ----------
 
-Perfekt! Das Entwickler-Forum unter http://jverein.de/forum/ ist dazu die erste Anlaufstelle. Das Entwickler-Forum ist nach Anmeldung und Freischaltung durch Heiner Jostkleigrewe verf�gbar. 
+Perfekt! Das Entwickler-Forum unter https://jverein-forum.de ist dazu die erste Anlaufstelle.
+Das Entwickler-Forum ist nach Anmeldung und Freischaltung durch Heiner Jostkleigrewe verfügbar.
+
+Um in die openjverein Organisation aufgenommen zu werden erstellt bitte ein issue.
 
 Sourcen
 -------
 
-JVerein nutzt die Homebankingsoftware Hibiscus und das GUI-Framework Jameica. F�r die Entwicklung m�ssen daher deren GIT-Repositorys eingebunden werden:
+JVerein nutzt die Homebankingsoftware Hibiscus und das GUI-Framework Jameica. Für die Entwicklung müssen daher deren GIT-Repositorys eingebunden werden:
 
    https://github.com/willuhn/jameica.git
```

Comment:
I proposing to use a bulleted list here for better layout:
- https://github.com/willuhn/jameica.git
- https://github.com/willuhn/hibiscus.git

Currently both repos are listed in a single line with only a space character separating them.